automation/01-install-rucio.bash

```diff
@@ -0,0 +1,285 @@
+echo "⟾ kubectl: Logs for Rucio server │"
+echo "└────────────────────────────────┘"
+kubectl logs "${RUCIO_SERVER_POD}" rucio-server
+
+echo "┌───────────────────────────────────┐"
+echo "⟾ kubectl: Install client container │"
+echo "└───────────────────────────────────┘"
+kubectl apply -f ../client.yaml
+while [[ "$(kubectl get pods client -o custom-columns=STATUS:.status.containerStatuses[*].ready --no-headers)" != true ]]; do
+  echo ""
+  kubectl get pods client
+  sleep 2
+done
+echo ""
+kubectl get pods client
+
+echo "┌─────────────────────────────────┐"
+echo "⟾ kubectl: Check client container │"
+echo "└─────────────────────────────────┘"
+kubectl exec client -it -- /etc/profile.d/rucio_init.sh
+kubectl exec client -it -- rucio whoami
+
+echo "┌─────────────────────────────────────────┐"
+echo "⟾ kubectl: Install XRootD storage systems │"
+echo "└─────────────────────────────────────────┘"
+kubectl apply -f ../xrd.yaml
+XRD_CONTAINERS=(xrd1 xrd2 xrd3)
+echo "XRD_CONTAINERS: ${XRD_CONTAINERS[*]}"
+for XRD_CONTAINER in "${XRD_CONTAINERS[@]}"; do
+  echo ""
+  echo "XRD_CONTAINER: ${XRD_CONTAINER}"
+  while [[ "$(kubectl get pods "${XRD_CONTAINER}" -o custom-columns=STATUS:.status.containerStatuses[*].ready --no-headers)" != true ]]; do
+    echo ""
+    kubectl get pods "${XRD_CONTAINER}"
+    sleep 2
+  done
+  echo ""
+  kubectl get pods "${XRD_CONTAINER}"
+done
+
+echo "┌───────────────────────────────────────┐"
+echo "⟾ kubectl: Install FTS database (MySQL) │"
+echo "└───────────────────────────────────────┘"
+kubectl apply -f ../ftsdb.yaml
+FTS_MYSQL_POD=$(kubectl get pods | grep ^fts-mysql- | awk '{print $1}')
+echo "FTS_MYSQL_POD: ${FTS_MYSQL_POD}"
+while [[ "$(kubectl get pods "${FTS_MYSQL_POD}" -o custom-columns=STATUS:.status.containerStatuses[*].ready --no-headers)" != true ]]; do
+  echo ""
+  kubectl get pods "${FTS_MYSQL_POD}"
+  sleep 1
+done
+echo ""
+kubectl get pods "${FTS_MYSQL_POD}"
+
+echo "┌────────────────────────────────────────┐"
+echo "⟾ kubectl: Logs for FTS database (MySQL) │"
+echo "└────────────────────────────────────────┘"
+kubectl logs "${FTS_MYSQL_POD}"
+
+echo "┌──────────────────────┐"
+echo "⟾ kubectl: Install FTS │"
+echo "└──────────────────────┘"
+kubectl apply -f ../fts.yaml
+FTS_SERVER_POD=$(kubectl get pods | grep ^fts-server- | awk '{print $1}')
+echo "FTS_SERVER_POD: ${FTS_SERVER_POD}"
+while [[ "$(kubectl get pods "${FTS_SERVER_POD}" -o custom-columns=STATUS:.status.containerStatuses[*].ready --no-headers)" != true ]]; do
+  echo ""
+  kubectl get pods "${FTS_SERVER_POD}"
+  sleep 1
+done
+echo ""
+kubectl get pods "${FTS_SERVER_POD}"
+
+echo "┌───────────────────────┐"
+echo "⟾ kubectl: Logs for FTS │"
+echo "└───────────────────────┘"
+kubectl logs "${FTS_SERVER_POD}"
+
+echo "┌───────────────────────┐"
+echo "⟾ helm: Install Daemons │"
+echo "└───────────────────────┘"
+echo "⤑ It will wait until the Daemons are set up. It could take several seconds or minutes."
```

Comment:
same

Reply:
Fixed in the commit 84bf9a925933eebb4b73482beb79825180033d7d.